code/modules/research/designs.dm

```diff
@@ -1374,6 +1374,14 @@ datum/design/item/weapon/ammo_9mm
 	materials = list("$metal" = 3750, "$silver" = 100)
 	build_path = /obj/item/ammo_magazine/c9mm
 
+datum/design/item/weapon/trod
+	id = "trod"
+	name = "tungsten rod pack"
+	desc = "moderately expensive superdense tungsten rods."
+	req_tech = list("combat" = 2, "materials" = 4)
+	materials = list("$metal" = 93750, "$gold" = 7500)

```

Comment:
Is it even possible to hold 93 750 units of metal in the RnD protolathe? Somewhere around 10 000 should be fine. For ref, 15 000 is the highest count I've been able to find.